packages/orderbook/test/js_api/vault.test.ts

```
@@ -309,4 +309,36 @@ describe('Rain Orderbook JS API Package Bindgen Vault Tests', async function ()
+	it('should handle approval amount less than allowance', async () => {
+		// Allowance is 100, and user tries to approve 90, so there should be approval calldata
+		await mockServer.forPost('/rpc').thenReply(
+			200,
+			JSON.stringify({
+				jsonrpc: '2.0',
+				id: 1,
+				result: '0x0000000000000000000000000000000000000000000000000000000000000064'
+			})
+		);
+
+		const res = await getVaultApprovalCalldata(mockServer.url + '/rpc', vault1, '90');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)